models/topic-model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+const format = require("pg-format");
+
+exports.fetchTopics = () => {
+  let query = "SELECT * FROM topics;";
```

Comment:
Should this be defined with a let? Do we need to edit this before we use it in db.query?